### ISSUE TITLE
[CV] Request for new CV entries

### ISSUE THREAD

**Issue by Mnago** [HUPO-PSI/mzQC#244]
### What is the QC term's name?

Peak Width, Retention Time of Targeted Peptide Precursors ,Precursor Ion Chromatogram ,Charge State Distribution of Identified Peptide Precursors, Intensity Variation of Adjacent MS1 Scans , Median Mass Accuracy of Identified Peptide Precursors , TIC MS1 Signal , Mass Accuracy of Identified Peptide Fragment Ions, MS2 Signal , : Ratios of Peak Area of MS1 to MS2 Signal , Number of Identified Peptides , Number of Identified Proteins , MS1 Area of Targeted Peptide Precursors , MS2 Intensities of Targeted Peptide Precursors,Ion Mobility Accuracy  

### Briefly describe the QC term.

Metric ID: F1 
Metric Name: Peak Width  
Description and Reason for Proposing: Peak width (FWHM) is used to evaluate column performance. A wider peak indicates decreased column performance, particularly for hydrophobic peptides. Sharp peaks result in higher signal intensities and a better signal-to-noise ratio, leading to an increased number of identified peptides.  
Metric Originates: Median peak width extracted from the DIA-NN main output with the column name “FWHM.scan.”
Value type: MS:4000003! single value

Metric ID: F2
Metric Name: Retention Time of Targeted Peptide Precursors  
Description and Reason for Proposing: Retention time measures the elution time and order of targeted peptide precursors. Fluctuations in peptide retention time can be used to assess issues such as liquid leakage or column performance.  
Metric Originates: Retention time of targeted peptide precursors extracted from the DIA-NN mainoutput with the column name “RT.”
Value type: MS:400004! n-tuple

Metric ID: F3  
Metric Name: Precursor Ion Chromatogram 
Description and Reason for Proposing: This metric provides comprehensive information regarding peptide elution, focusing on both hydrophilic and hydrophobic peptides.  
Metric Originates: By converting the raw file to mzXML, decrypting it to obtain MS1 signals at each time point, and then performing linear interpolation to generate the final chromatogram.
Value type: MS:400005! table

Metric ID: F4 
Metric Name: Charge State Distribution of Identified Peptide Precursors
Description and Reason for Proposing: This metric is crucial for peptide identification. The distribution of charge states can vary based on peptide length, composition, and ionization conditions. Monitoring charge state distribution helps assess the quality of peptide identification and identify potential issues affecting result accuracy.  
Metric Originates: This metric is derived from the statistical analysis of charge state distributions of peptide ions identified by DIA-NN, categorized into +1, +2, +3, +4, +5, and +6 charges.
Value type: MS:400004! n-tuple

Metric ID: F5 
Metric Name: Intensity Variation of Adjacent MS1 Scans 
Description and Reason for Proposing: This metric evaluates electrospray stability, a critical factor for reliable and reproducible mass spectra. By analyzing the ratio of precursor ion current for adjacent MS1 scans and counting the frequency of MS1 intensity changes, issues with electrospray stability can be identified and addressed. Factors such as sample quality, spray needle condition, and ion source voltage can affect electrospray stability, leading to inconsistent results.  
Metric Originates: This metric is obtained by converting the raw file to mzXML, decrypting it to get MS1 signal data for each time point, and then calculating the variation in adjacent MS1 scans.
Value type: MS:4000003! single value

Metric ID: F6  
Metric Name: Median Mass Accuracy of Identified Peptide Precursors  
Description and Reason for Proposing: This metric describes the mass accuracy of MS1 spectra. Factors such as temperature, airflow, or instrument calibration can cause mass shifts. Deviations in mass accuracy affect peptide identification and quantification. Therefore, controlling mass accuracy is essential.  
Metric Originates: Extracted from the DIA-NN main output stats file with the column name Median.Mass.Acc.MS1.
Value type: MS:4000003! single value

Metric ID: F7
Metric Name: TIC MS1 Signal 
Description and Reason for Proposing: This metric represents the total ion current (TIC) for MS1 chromatogram profiling. The signal can be affected by contamination, retention time shifts, or loss of hydrophilic/hydrophobic peptides. Comparisons between raw files from different instrument types may be limited, but data from the same instruments can be used to assess consistency.  
Metric Originates: Extracted from the DIA-NN main output stats file with the column name “MS1.Signal.”
Value type: MS:4000003! single value

Metric ID: F8
Metric Name: Mass Accuracy of Identified Peptide Fragment Ions
Description and Reason for Proposing: This metric describes the mass accuracy of MS2 spectra. Factors like temperature, airflow, or instrument calibration can cause mass shifts. Deviations in mass accuracy affect peptide identification and quantification, so it’s important to monitor MS2 mass accuracy.  
Metric Originates: Extracted from the DIA-NN main output stats file with the column name Median.Mass.Acc.MS2.
Value type: MS:4000003! single value

Metric ID: F9  
Metric Name: MS2 Signal  
Description and Reason for Proposing: Issues such as contamination on the MS front-end and fragmentation efficiency can lead to poor MS2 signals. Contamination would decrease both MS1 and MS2 signals, while poor fragmentation efficiency affects MS2 signal but leaves MS1 signal unchanged.  
Metric Originates: Extracted from the DIA-NN main output stats file with the column name “MS2.Signal.”
Value type: MS:4000003! single value

Metric ID: F10
Metric Name: Ratios of Peak Area of MS1 to MS2 Signal 
Description and Reason for Proposing: This metric is calculated by dividing the MS1 signal peak area by the MS2 signal peak area. Monitoring this ratio can help distinguish between MS front-end contamination and decreased ion fragmentation capability when features F7 and F9 are not identifiable.  
Metric Originates: Calculated based on the ratio of metrics F7 and F9.
Value type: MS:4000003! single value

Metric ID: F11  
Metric Name: Number of Identified Peptides 
Description and Reason for Proposing: This metric represents the number of peptides identified by matching MS2 spectra to those in a spectral library. Various factors, such as peak width, retention time distribution, TIC MS1 signal, and TIC MS2 signal, can influence this number. It is a critical feature for evaluating spectral quality.  
Metric Originates: Extracted from the DIA-NN main output file, reflecting the count of identified peptides.
Value type: MS:4000003! single value

Metric ID: F12 
Metric Name: Number of Identified Proteins  
Description and Reason for Proposing: This metric indicates the number of proteins identified through peptide-to-protein matching. Factors affecting peptide identification also impact protein identification. Therefore, the number of identified proteins is an important feature for assessing raw file quality.  
Metric Originates: Extracted from the DIA-NN main output file, reflecting the count of identified proteins.
Value type: MS:4000003! single value

Metric ID: F13
Metric Name: MS1 Area of Targeted Peptide Precursors 
Description and Reason for Proposing: This metric presents the MS1 area of targeted peptide precursors. It helps evaluate the MS1 area of peptide precursors selected through machine learning, providing a good assessment of peptide quality in the mass spectrometry data.  
Metric Originates: Data extracted from the DIA-NN main output file, indicating MS1 area.
Value type: MS:400004! n-tuple

Metric ID: F14 
Metric Name: MS2 Intensities of Targeted Peptide Precursors
Description and Reason for Proposing: This metric presents the MS2 intensities of targeted peptide precursors. It helps evaluate the MS2 intensities of peptides selected through machine learning, providing a good assessment of peptide quality in the mass spectrometry data.  
Metric Originates: Data extracted from the DIA-NN main output file, indicating MS2 intensity.
Value type: MS:400004! n-tuple

Metric ID: F15  
Metric Name: Ion Mobility Accuracy  
Description and Reason for Proposing: This metric reports the ion mobility accuracy of a standard sample with an m/z of 622.0290 and reduced ion mobility coefficients (1/K0) of 0.9915, extracted from diaPASEF files. This standard sample, the ESI-L low concentration tuning mix from Agilent, was spiked into the ion source filter of the timsTOF Pro instrument. This metric is specifically used for raw files generated by timsTOF Pro instruments. Ion mobility accuracy directly affects the precision of ion release in diaPASEF acquisition, impacting peptide ion identification and quantification.  
Metric Originates: Extracted from timsTOF generated MGF files, measuring ion mobility values for the standard sample m/z of 622.0290 within 0.3 minutes before peak appearance to assess accuracy.
Value type: MS:4000003! single value

### Value type

MS:4000003 ! single value

**Comment by cbielow** [comment]
thanks for proposing new terms.
We need to cross-check with existing terms to ensure that there are no duplicates. 

On first sight there is:

for 'F1 Peak Width':
[FWHM](https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L794), even though this is a single value with to context. But [XIC-FWHM quantiles](https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L22711) may be a good match. However, you'd need to come up with 2 more values to make this metric work. Filling it with NA's may be valid?

for `F12 Number of Identified Proteins`:
[number of identified protein groups](https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L21976)
there is also
[count of identified clusters](https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L15846), which may already be a duplicate?

to be continued.

If you want to have a look yourself, that would certainly be helpful in speeding up this process.
There are, however, certainly new metrics in here, which will enrich the CV, so thanks for sharing.

**Comment by Mnago** [comment]
Thank you for your reply. I have already reviewed the metrics myself and kept only the non-overlapping ones in the metric library. Among the retained metrics, there is one that I have doubts about regarding whether they overlap, but I believe the rest do not overlap. I have provided my suggestions.

Thank you for your help, and I hope to receive the term IDs for these metrics soon.
---

Metric ID: F2 (Confused)
Metric Name: Retention Time of Targeted Peptide Precursors  
Description and Reason for Proposing: Retention time measures the elution time and order of targeted peptide precursors. Fluctuations in peptide retention time can be used to assess issues such as liquid leakage or column performance.  
Metric Originates: Retention time of targeted peptide precursors extracted from the DIA-NN mainoutput with the column name “RT.”
Value type: MS:400004! n-tuple


[Term]
id: MS:1002064
name: peptide consensus RT
def: "Peptide consensus retention time." [PSI:PI]
is_a: MS:1002737 ! peptide-level quantification datatype
relationship: has_metric_category MS:4000016 ! retention time metric
relationship: has_value_type xsd:double ! The allowed value-type for this CV term


Our metric, "retention time of targeted peptide precursors," seems to overlap with an existing metric named "peptide consensus RT." Could you please help confirm whether the two represent the same concept?
---

Metric ID: F3  
Metric Name: Precursor Ion Chromatogram 
Description and Reason for Proposing: This metric provides comprehensive information regarding peptide elution, focusing on both hydrophilic and hydrophobic peptides.  
Metric Originates: By converting the raw file to mzXML, decrypting it to obtain MS1 signals at each time point, and then performing linear interpolation to generate the final chromatogram. The reason for this approach is to normalize MS1 scans from instruments with any scanning speed to the same level for analysis. After processing in this manner, each file's metric contains 1000 dimensions of data for description.
Value type: MS:400005! Table

According to my matching, this feature does not correspond to any previous metric ID. I suggest creating a new metric. 
---

Metric ID: F4 
Metric Name: Charge State Distribution of Identified Peptide Precursors
Description and Reason for Proposing: This metric is crucial for peptide identification. The distribution of charge states can vary based on peptide length, composition, and ionization conditions. Monitoring charge state distribution helps assess the quality of peptide identification and identify potential issues affecting result accuracy.  
Metric Originates: This metric is derived from the statistical analysis of charge state distributions of peptide ions identified by DIA-NN, categorized into +1, +2, +3, +4, +5, and +6 charges.
Value type: MS:400004! n-tuple

This metric assesses the charge distribution trends of all identified peptide precursor ions, including +1, +2, +3, +4, +5, and +6 charges. This metric differs from existing charge-related metrics. For example, IDs MS:1001705-1001707 evaluate the std high confidence X Corr charge, which does not include higher charge states such as +5 and +6. Additionally, IDs MS:4000168 to MS:4000176 are used to evaluate the charge states of fragment ions. Therefore, we propose creating a new metric ID.
---

Metric ID: F6  
Metric Name: Median Mass Accuracy of Identified Peptide Precursors  
Description and Reason for Proposing: This metric describes the mass accuracy of MS1 spectra. Factors such as temperature, airflow, or instrument calibration can cause mass shifts. Deviations in mass accuracy affect peptide identification and quantification. Therefore, controlling mass accuracy is essential.  
Metric Originates: Extracted from the DIA-NN main output stats file with the column name Median.Mass.Acc.MS1.
Value type: MS:4000003! single value

This metric assesses the median mass accuracy of all identified peptide fragment ions. ID MS:4000178 represents the ppm deviation of identified peptide precursors, so a new metric ID is needed.
---

Metric ID: F7
Metric Name: TIC MS1 Signal 
Description and Reason for Proposing: This metric represents integral area of the total ion current (TIC) for MS1 chromatogram profiling. The signal can be affected by contamination, retention time shifts, or loss of hydrophilic/hydrophobic peptides. Comparisons between raw files from different instrument types may be limited, but data from the same instruments can be used to assess consistency.  
Metric Originates: Extracted from the DIA-NN main output stats file with the column name “MS1.Signal.”
Value type: MS:4000003! single value

This metric represents the area obtained by integrating the contour plot formed by the ion chromatogram and the X-axis. This metric does not overlap with any existing metrics, so a new metric needs to be defined.
---

Metric ID: F8
Metric Name: Mass Accuracy of Identified Peptide Fragment Ions
Description and Reason for Proposing: This metric describes the mass accuracy of MS2 spectra. Factors like temperature, airflow, or instrument calibration can cause mass shifts. Deviations in mass accuracy affect peptide identification and quantification, so it’s important to monitor MS2 mass accuracy.  
Metric Originates: Extracted from the DIA-NN main output stats file with the column name Median.Mass.Acc.MS2.
Value type: MS:4000003! single value

This metric assesses the median mass accuracy of all identified peptide fragment ions. Existing metrics include ID MS:4000072, which represents observed mass accuracy, and ID MS:4000178, which represents the ppm deviation of identified peptide precursors. Therefore, a new metric ID is needed. The recommendation is as follows.
---

Metric ID: F9  
Metric Name: MS2 Signal  
Description and Reason for Proposing: Issues such as contamination on the MS front-end and fragmentation efficiency can lead to poor MS2 signals. Contamination would decrease both MS1 and MS2 signals, while poor fragmentation efficiency affects MS2 signal but leaves MS1 signal unchanged.  
Metric Originates: Extracted from the DIA-NN main output stats file with the column name “MS2.Signal.”
Value type: MS:4000003! single value

This metric represents the sum of the areas obtained by integrating the contour plot formed by the fragment ion chromatogram and the X-axis. This metric does not overlap with any existing metrics, so a new metric needs to be defined.
---

Metric ID: F10
Metric Name: Ratios of Peak Area of MS1 to MS2 Signal 
Description and Reason for Proposing: This metric is calculated by dividing the MS1 signal peak area by the MS2 signal peak area. Monitoring this ratio can help distinguish between MS front-end contamination and decreased ion fragmentation capability when metrics TIC MS1 signal and TIC MS2 signal are not identifiable.  
Metric Originates: Calculated based on the ratio of TIC MS1 signal and TIC MS2 signal.
Value type: MS:4000003! single value

This metric is obtained by integrating the area ratio of the contour plots formed by the ion current profiles of the parent ion and fragment ions on the X-axis. This metric does not overlap with any existing metrics, so a new metric needs to be redefined.
---

The above are the metrics that I personally believe need to be added.

I look forward to your reply. Thanks!

Best regards,

Winnie

**Comment by cbielow** [comment]
> Hello,
> 
> Thank you for your reply. I have already reviewed the metrics myself and kept only the non-overlapping ones in the metric library. Among the retained metrics, there are two that I have doubts about regarding whether they overlap, but I believe the rest do not overlap. I have provided my suggestions.
> 
> Metric ID: F2 (Confused) Metric Name: Retention Time of Targeted Peptide Precursors Description and Reason for Proposing: Retention time measures the elution time and order of targeted peptide precursors. Fluctuations in peptide retention time can be used to assess issues such as liquid leakage or column performance. Metric Originates: Retention time of targeted peptide precursors extracted from the DIA-NN mainoutput with the column name “RT.” Value type: MS:400004! n-tuple
> 
> [Term] id: MS:1002064 name: peptide consensus RT def: "Peptide consensus retention time." [PSI:PI] is_a: MS:1002737 ! peptide-level quantification datatype relationship: has_metric_category MS:4000016 ! retention time metric relationship: has_value_type xsd:double ! The allowed value-type for this CV term
> 
> Our metric, "retention time of targeted peptide precursors," seems to overlap with an existing metric named "peptide consensus RT." Could you please help confirm whether the two represent the same concept?

My interpretation of "peptide consensus RT" would be an average/aggregate of multiple RTs of the same peptide across multiple samples, i.e. what is commonly referred to as 'match-between-runs', after an RT alignment between multiple runs. So this is probably not what you want.
There is a very generic
```
[Term]
id: MS:1000894
name: retention time
```
which you might want to use or derive from (using 'is_a'), see `MS:1000895` for an example.


> 
> Metric ID: F3 Metric Name: Precursor Ion Chromatogram Description and Reason for Proposing: This metric provides comprehensive information regarding peptide elution, focusing on both hydrophilic and hydrophobic peptides. Metric Originates: By converting the raw file to mzXML, decrypting it to obtain MS1 signals at each time point, and then performing linear interpolation to generate the final chromatogram. The reason for this approach is to normalize MS1 scans from instruments with any scanning speed to the same level for analysis. After processing in this manner, each file's metric contains 1000 dimensions of data for description. Value type: MS:400005! Table
> 
> According to my matching, this feature does not correspond to any previous metric ID. I suggest creating a new metric.

I'm not sure I understand what is in this table (can you add column descriptions, see https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L23347).
How would this metric be different from a TIC? (see the `MS:4000104` term I linked above)


> 
> Metric ID: F4 Metric Name: Charge State Distribution of Identified Peptide Precursors Description and Reason for Proposing: This metric is crucial for peptide identification. The distribution of charge states can vary based on peptide length, composition, and ionization conditions. Monitoring charge state distribution helps assess the quality of peptide identification and identify potential issues affecting result accuracy. Metric Originates: This metric is derived from the statistical analysis of charge state distributions of peptide ions identified by DIA-NN, categorized into +1, +2, +3, +4, +5, and +6 charges. Value type: MS:400004! n-tuple
> 
> This metric assesses the charge distribution trends of all identified peptide precursor ions, including +1, +2, +3, +4, +5, and +6 charges. This metric differs from existing charge-related metrics. For example, IDs MS:1001705-1001707 evaluate the std high confidence X Corr charge, which does not include higher charge states such as +5 and +6. Additionally, IDs MS:4000168 to MS:4000176 are used to evaluate the charge states of fragment ions. Therefore, we propose creating a new metric ID.

`MS:4000063` and/or `MS:4000064` should cover this?

> 
> Metric ID: F6 Metric Name: Median Mass Accuracy of Identified Peptide Precursors Description and Reason for Proposing: This metric describes the mass accuracy of MS1 spectra. Factors such as temperature, airflow, or instrument calibration can cause mass shifts. Deviations in mass accuracy affect peptide identification and quantification. Therefore, controlling mass accuracy is essential. Metric Originates: Extracted from the DIA-NN main output stats file with the column name Median.Mass.Acc.MS1. Value type: MS:4000003! single value
> 
> This metric assesses the median mass accuracy of all identified peptide fragment ions. 

Your description uses 'Precursors' and 'MS1'. So this is meant as a mass accuracy for the precursor in MS1, right?

>ID MS:4000178 represents the ppm deviation of identified peptide precursors, so a new metric ID is needed.

So I'd think that this is a very good match, but since this reports the mean and you require median, the `MS:4000178` can be copied and adapted for this.



> 
> Metric ID: F7 Metric Name: TIC MS1 Signal Description and Reason for Proposing: This metric represents integral area of the total ion current (TIC) for MS1 chromatogram profiling. The signal can be affected by contamination, retention time shifts, or loss of hydrophilic/hydrophobic peptides. Comparisons between raw files from different instrument types may be limited, but data from the same instruments can be used to assess consistency. Metric Originates: Extracted from the DIA-NN main output stats file with the column name “MS1.Signal.” Value type: MS:4000003! single value
> 
> This metric represents the area obtained by integrating the contour plot formed by the ion chromatogram and the X-axis. This metric does not overlap with any existing metrics, so a new metric needs to be defined.

I think `id: MS:4000155` (area under TIC) is a very good match.


> 
> Metric ID: F8 Metric Name: Mass Accuracy of Identified Peptide Fragment Ions Description and Reason for Proposing: This metric describes the mass accuracy of MS2 spectra. Factors like temperature, airflow, or instrument calibration can cause mass shifts. Deviations in mass accuracy affect peptide identification and quantification, so it’s important to monitor MS2 mass accuracy. Metric Originates: Extracted from the DIA-NN main output stats file with the column name Median.Mass.Acc.MS2. Value type: MS:4000003! single value
> 
> This metric assesses the median mass accuracy of all identified peptide fragment ions. Existing metrics include ID MS:4000072, which represents observed mass accuracy, and ID MS:4000178, which represents the ppm deviation of identified peptide precursors. Therefore, a new metric ID is needed. The recommendation is as follows.
> 

Indeed. Can also be an analog to `MS:4000178`. Maybe we should add the mean (as opposed to the median required here) and standard deviation while we are at it (see `MS:4000179`).



> Metric ID: F9 Metric Name: MS2 Signal Description and Reason for Proposing: Issues such as contamination on the MS front-end and fragmentation efficiency can lead to poor MS2 signals. Contamination would decrease both MS1 and MS2 signals, while poor fragmentation efficiency affects MS2 signal but leaves MS1 signal unchanged. Metric Originates: Extracted from the DIA-NN main output stats file with the column name “MS2.Signal.” Value type: MS:4000003! single value
> 
> This metric represents the sum of the areas obtained by integrating the contour plot formed by the fragment ion chromatogram and the X-axis. This metric does not overlap with any existing metrics, so a new metric needs to be defined.

So this is basically an MS2 TIC (analog to `MS:4000155`)? I could not find a match in the current CV. Anyone else? @bittremieux @mwalzer?


> 
> Metric ID: F10 Metric Name: Ratios of Peak Area of MS1 to MS2 Signal Description and Reason for Proposing: This metric is calculated by dividing the MS1 signal peak area by the MS2 signal peak area. Monitoring this ratio can help distinguish between MS front-end contamination and decreased ion fragmentation capability when metrics TIC MS1 signal and TIC MS2 signal are not identifiable. Metric Originates: Calculated based on the ratio of TIC MS1 signal and TIC MS2 signal. Value type: MS:4000003! single value
> 
> This metric is obtained by integrating the area ratio of the contour plots formed by the ion current profiles of the parent ion and fragment ions on the X-axis. This metric does not overlap with any existing metrics, so a new metric needs to be redefined.

Interesting concept! Most certainly non-existant. So this simply divides F7 / F9?



Thanks for the patience :)

**Comment by Mnago** [comment]
Hi,

Thank you for your response. To expedite the process, I will reply individually to the metrics that need further discussion and handling.


Metric ID: F3 Metric Name: Precursor Ion Chromatogram Description and Reason for Proposing: This metric provides comprehensive information regarding peptide elution, focusing on both hydrophilic and hydrophobic peptides. Metric Originates: By converting the raw file to mzXML, decrypting it to obtain MS1 signals at each time point, and then performing linear interpolation to generate the final chromatogram. The reason for this approach is to normalize MS1 scans from instruments with any scanning speed to the same level for analysis. After processing in this manner, each file's metric contains 1000 dimensions of data for description. Value type: MS:400005! Table
I'm not sure I understand what is in this table (can you add column descriptions, see https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L23347).
How would this metric be different from a TIC? (see the MS:4000104 term I linked above)

Reply: This metric pertains to information from all the parent ion chromatograms, not the total ion chromatogram. Therefore, I still recommend providing a new term. Compared to the MS: 4000104 term, I believe this feature is relatively closer to the MS: 1000235 term. Therefore, based on my understanding, I have uploaded the new information at the end to the website you recommended. The details are as follows.
[Term]
id: MS:1003413
name: precursor ion current chromatogram
def: "Representation of the precursor ion current detected in each a series of MS1 spectra versus time." [PSI:MS]
synonym: "precursor ion chromatogram" EXACT [PSI:MS]
is_a: MS:1000810 ! ion current chromatogram



Metric ID: F8 Metric Name: Mass Accuracy of Identified Peptide Fragment Ions Description and Reason for Proposing: This metric describes the mass accuracy of MS2 spectra. Factors like temperature, airflow, or instrument calibration can cause mass shifts. Deviations in mass accuracy affect peptide identification and quantification, so it’s important to monitor MS2 mass accuracy. Metric Originates: Extracted from the DIA-NN main output stats file with the column name Median.Mass.Acc.MS2. Value type: MS:4000003! single value

This metric assesses the median mass accuracy of all identified peptide fragment ions. Existing metrics include ID MS:4000072, which represents observed mass accuracy, and ID MS:4000178, which represents the ppm deviation of identified peptide precursors. Therefore, a new metric ID is needed. The recommendation is as follows.

Indeed. Can also be an analog to MS:4000178. Maybe we should add the mean (as opposed to the median required here) and standard deviation while we are at it (see MS:4000179).

Reply: Yes, you can add "median" to MS:4000179.





Metric ID: F9 Metric Name: MS2 Signal Description and Reason for Proposing: Issues such as contamination on the MS front-end and fragmentation efficiency can lead to poor MS2 signals. Contamination would decrease both MS1 and MS2 signals, while poor fragmentation efficiency affects MS2 signal but leaves MS1 signal unchanged. Metric Originates: Extracted from the DIA-NN main output stats file with the column name “MS2.Signal.” Value type: MS:4000003! single value
This metric represents the sum of the areas obtained by integrating the contour plot formed by the fragment ion chromatogram and the X-axis. This metric does not overlap with any existing metrics, so a new metric needs to be defined.
So this is basically an MS2 TIC (analog to MS:4000155)? I could not find a match in the current CV. Anyone else? @bittremieux @mwalzer?

Reply: Yes, this actually refers to the "area of All MS2 chromatogram." If you haven't found a matching term, you might consider creating a new term ID.



Metric ID: F10 Metric Name: Ratios of Peak Area of MS1 to MS2 Signal Description and Reason for Proposing: This metric is calculated by dividing the MS1 signal peak area by the MS2 signal peak area. Monitoring this ratio can help distinguish between MS front-end contamination and decreased ion fragmentation capability when metrics TIC MS1 signal and TIC MS2 signal are not identifiable. Metric Originates: Calculated based on the ratio of TIC MS1 signal and TIC MS2 signal. Value type: MS:4000003! single value
This metric is obtained by integrating the area ratio of the contour plots formed by the ion current profiles of the parent ion and fragment ions on the X-axis. This metric does not overlap with any existing metrics, so a new metric needs to be redefined.
Interesting concept! Most certainly non-existant. So this simply divides F7 / F9?

Reply: Yes, this is simply F7/F9. Could you please add a new term ID?

Best regards,

Winnie

**Comment by cbielow** [comment]
ok, to summarize:


`F3`:

```
[Term]
id: MS:1000????
name: precursor ion current chromatogram
def: "Representation of the ion current assigned to detected precursors in the series of all MS1 spectra versus time." [PSI:MS]
synonym: "precursor ion chromatogram" EXACT []
is_a: MS:1000810 ! ion current chromatogram
```
Note that this not a table, but rather just an n-tuple (with as many values as there are MS1 spectra).
If you want/need a more elaborate mapping (e.g. to native identifier) for each MS1 spectrum, you'd need a table.



`F8`:
```
[Term]
id: MS:400????
name: fragment ppm deviation median
def: "The median of the distribution of observed fragment mass accuracies (MS:4000072) [in ppm] of identified MS2 spectra after user-defined acceptance criteria (FDR) are applied" [PSI:MS]
is_a: MS:4000003 ! single value
relationship: has_metric_category MS:4000008 ! ID based metric
relationship: has_metric_category MS:4000022 ! MS2 metric
relationship: has_value_concept MS:1000014 ! accuracy
relationship: has_units UO:0000169 ! parts per million
relationship: has_value_type xsd:float ! The allowed value-type for this CV term
relationship: has_value_concept STATO:0000574 ! center value
```
and in addition (for completeness since we already have `precursor ppm deviation mean` + sigma):
```
[Term]
id: MS:400????
name: fragment ppm deviation mean
def: "The mean of the distribution of observed fragment mass accuracies (MS:4000072) [in ppm] of identified MS2 spectra after user-defined acceptance criteria (FDR) are applied" [PSI:MS]
is_a: MS:4000003 ! single value
relationship: has_metric_category MS:4000008 ! ID based metric
relationship: has_metric_category MS:4000022 ! MS2 metric
relationship: has_value_concept MS:1000014 ! accuracy
relationship: has_units UO:0000169 ! parts per million
relationship: has_value_type xsd:float ! The allowed value-type for this CV term
relationship: has_value_concept STATO:0000401 ! sample mean

[Term]
id: MS:400????
name: fragment ppm deviation sigma
def: "The standard deviation of the distribution of observed fragment mass accuracies (MS:4000072) [in ppm] of identified MS2 spectra after user-defined acceptance criteria (FDR) are applied" [PSI:MS]
is_a: MS:4000003 ! single value
relationship: has_metric_category MS:4000008 ! ID based metric
relationship: has_metric_category MS:4000022 ! MS2 metric
relationship: has_value_concept MS:1000830 ! precision
relationship: has_units UO:0000169 ! parts per million
relationship: has_value_type xsd:float ! The allowed value-type for this CV term
relationship: has_value_concept STATO:0000237 ! standard deviation
```

`F9`
```
[Term]
id: MS:4000???
name: area under TIC in MS2
def: "The area under the total ion chromatogram of all MS2 spectra." [PSI:MS]
comment: The metric informs about the dynamic range of the acquisition. Differences between samples of an experiment may indicate differences in the dynamic range and/or in the sample content.
is_a: MS:4000003 ! single value
relationship: has_metric_category MS:4000009 ! ID free
relationship: has_metric_category MS:4000017 ! chromatogram metric
```

`F10`
```
[Term]
id: MS:100???
name: Ratio of Peak Area of MS1 vs MS2 Signal 
def: "The ratio of the areas under TIC of MS1 (MS:1000235) divided by the area under the TIC of MS2 (MS:4000??? -- F9 todo!)." [PSI:MS]
is_a: MS:4000003 ! single value
is_a: MS:1001848 ! simple ratio of two values
relationship: has_value_type xsd:float ! The allowed value-type for this CV term
```

@bittremieux @mwalzer  @nilshoffmann  I'm not sure if MS:1000235 is actually only meant to be based on MS1. Could very well mean all MS scans, irrespective of MS level. Should we introduce a new term for MS1 explicitly?


@Mnago
Does this sound reasonable?
What about the other metrics? E.g. F2?

**Comment by Mnago** [comment]
Hi,

Thank you for your prompt response.

I agree with your additions to the descriptions for F3, F8, F9, and F10. For F8, I still prefer defining it as "name: fragment ppm deviation median." 

I’m not sure if I should add the IDs for F3, F8, F9, and F10, or if that’s something you will handle. Let me know if there’s anything you need me to do; I’m happy to assist.

Additionally, I have used the ID MS:1000894, as you recommended previously, to replace F2.

Looking forward to your response.

Best regards,

Winnie

**Comment by bittremieux** [comment]
> I'm not sure if MS:1000235 is actually only meant to be based on MS1. Could very well mean all MS scans, irrespective of MS level. Should we introduce a new term for MS1 explicitly?

Indeed, I think this is a more general term that can refer to any MS scan.

**Comment by Mnago** [comment]
Hi,

Thank you, Bittremieux,

Could I update the new CV term at the following link? https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L23347

Best regards,

Winnie

**Comment by cbielow** [comment]
No, the changes in the pull request (https://github.com/HUPO-PSI/psi-ms-CV/pull/333) will add the terms.
It's almost ready. Stay tuned :)

**Comment by bittremieux** [comment]
The new terms are now included in the PSI-MS CV version 4.1.179 and up. See accession `MS:4000025` to `MS:4000031`. 🙂
